RecoJets/JetProducers/python/PileupJetIDParams_cfi.py

```diff
@@ -163,6 +163,19 @@
 full_106x_UL17_chs.trainings[3].tmvaWeights = "RecoJets/JetProducers/data/pileupJetId_UL17_Eta3p0To5p0_chs_BDT.weights.xml.gz"
 full_106x_UL17_chs.trainings[3].tmvaVariables = trainingVariables_106X_Eta3To5
 
+####################################################################################################################
+trainingVariables_106X_Eta0To3 = list(trainingVariables_102X_Eta0To3)
+trainingVariables_106X_Eta3To5 = list(trainingVariables_102X_Eta3To5)
+full_106x_UL18_chs = full_81x_chs.clone(JetIdParams = full_106x_UL18_chs_wp)
+full_106x_UL18_chs.trainings[0].tmvaWeights = "RecoJets/JetProducers/data/pileupJetId_UL18_Eta0p0To2p5_chs_BDT.weights.xml.gz"
+full_106x_UL18_chs.trainings[0].tmvaVariables = trainingVariables_106X_Eta0To3
+full_106x_UL18_chs.trainings[1].tmvaWeights = "RecoJets/JetProducers/data/pileupJetId_UL18_Eta2p5To2p75_chs_BDT.weights.xml.gz"
+full_106x_UL18_chs.trainings[1].tmvaVariables = trainingVariables_106X_Eta0To3
+full_106x_UL18_chs.trainings[2].tmvaWeights = "RecoJets/JetProducers/data/pileupJetId_UL18_Eta2p75To3p0_chs_BDT.weights.xml.gz"
+full_106x_UL18_chs.trainings[2].tmvaVariables = trainingVariables_106X_Eta0To3
+full_106x_UL18_chs.trainings[3].tmvaWeights = "RecoJets/JetProducers/data/pileupJetId_UL18_Eta3p0To5p0_chs_BDT.weights.xml.gz"
+full_106x_UL18_chs.trainings[3].tmvaVariables = trainingVariables_106X_Eta3To5
```

Comment:
```suggestion
full_106x_UL18_chs = full_106x_UL17_chs.clone(JetIdParams = full_106x_UL18_chs_wp)
for train in full_106x_UL18_chs.trainings: train.tmvaWeights = train.tmvaWeights.value().replace("UL17", "UL18")
```
seems to be more compact

Reply:
> > New training files (UL18) and configs are added for Pileup Jet Id (AK4CHS), presented [here](https://indico.cern.ch/event/973155/#69-ul18-pu-jet-id-wp-update)
> 
> e.g. on p8
> <img alt="Screen Shot 2020-12-17 at 5 07 20 PM" width="488" src="https://user-images.githubusercontent.com/4676718/102561778-6a6b1800-408a-11eb-961c-2b90e2cb29b9.png">
> please clarify what is used for "Old". Is this following the current baseline in 11_2_X/11_3_X where `_chsalgos_106X_UL17` is applied to everything, or is this the 106X kind of old where apparently `_chsalgos_81x` is applied?

This is the MINIAOD sample `/DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer19UL18MiniAOD-106X_upgrade2018_realistic_v11_L1v1-v1/MINIAODSIM`,  the old refers to the pileup ID value stored in it.

This dataset was produced with `CMSSW_10_6_4_patch1` , after digging through code [1], I can confirm the `old` here is indeed `_chsalgos_81x`.

[1] https://github.com/cms-sw/cmssw/blob/CMSSW_10_6_4_patch1/RecoJets/JetProducers/python/PileupJetID_cfi.py#L15